tests/run-pass/portable-simd.rs

```diff
@@ -20,7 +20,9 @@ fn simd_ops_i32() {
     assert_eq!(a * b, i32x4::from_array([10, 20, 30, 40]));
     assert_eq!(a / b, i32x4::from_array([10, 5, 3, 2]));
     assert_eq!(a / i32x4::splat(2), i32x4::splat(5));
+    assert_eq!(i32x2::splat(i32::MIN) / i32x2::splat(-1), i32x2::splat(i32::MIN));
     assert_eq!(a % b, i32x4::from_array([0, 0, 1, 2]));
+    assert_eq!(i32x2::splat(i32::MIN) % i32x2::splat(-1), i32x2::splat(0));
```

Comment:
Interestingly, this is allowed now in portable SIMD, but panics for regular scalar division / remainder.
Cc @workingjubilee